src/cache/precipitation_precomputed_quantities.jl

```diff
@@ -112,6 +112,11 @@ function set_precipitation_velocities!(
     cm2p = CAP.microphysics_2m_params(p.params)
     thp = CAP.thermodynamics_params(p.params)
 
+    q_liq = ᶜspecific(Y.c.ρq_liq, Y.c.ρ)
+    q_ice = ᶜspecific(Y.c.ρq_ice, Y.c.ρ)
+    q_rai = ᶜspecific(Y.c.ρq_rai, Y.c.ρ)
+    q_sno = ᶜspecific(Y.c.ρq_sno, Y.c.ρ)
+
```

Comment:
Do we need to define q_liq etc. here? They are re-computed in the calls to the terminal velocities anyway.